<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tanchuang</class>
 <widget class="QDialog" name="tanchuang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>218</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>62</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;

font-weight:400;
color:rgba(40,40,40,1);
line-height:34px;
opacity:0.97;
</string>
    </property>
    <property name="text">
     <string>Are you sure to exit the note book?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:14px;
font-weight:400;
color:rgba(102,102,102,1);
line-height:40px;
opacity:0.91;</string>
    </property>
    <property name="text">
     <string>Close the desktop note page at the same time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>166</y>
      <width>120</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(102,102,102,1);
line-height:24px;</string>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>166</y>
      <width>120</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(102,102,102,1);
line-height:24px;</string>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
